src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -23,10 +24,12 @@
 class DumpExtension extends \Twig_Extension
 {
     private $cloner;
+    private $dumper;
 
-    public function __construct(ClonerInterface $cloner)
+    public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper = null)
```

Comment:
what about injecting the dumper as last argument of the dump method instead? this would be even more flexibile, by allowing one to dynamically reconfigure the dumper at runtime, don't you think?